schemas/hygrothermalData.json

```diff
@@ -1,37 +1,183 @@
+        },
+        "moistureStorageFunction": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "relativeHumidityAbsolut": {
+                "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                "description": "short description"
+              },
+              "waterContent": {
+                "$ref": "number.json#/$defs/kilogramPerCubicMeter",
+                "description": "short description"
+              }
             }
-          },
-          "additionalProperties": false,
-          "minProperties": 1,
-          "required": ["xmlData"]
+          }
+        },
+        "moistureStorageFunctionGenerated": {
+          "type": "boolean"
+        },
+        "liquidTransportCoefficientSuction": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "waterContent": {
+                "$ref": "number.json#/$defs/kilogramPerCubicMeter",
+                "description": "short description"
+              },
+              "liquidTransportCoefficient": {
+                "$ref": "number.json#/$defs/squareMeterPerSecond",
+                "description": "short description"
+              }
+            }
+          }
+        },
+        "liquidTransportCoefficientSuctionGenerated": {
+          "type": "boolean"
+        },
+        "liquidTransportCoefficientRedistribution": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "waterContent": {
+                "$ref": "number.json#/$defs/kilogramPerCubicMeter",
+                "description": "short description"
+              },
+              "liquidTransportCoefficient": {
+                "$ref": "number.json#/$defs/squareMeterPerSecond",
+                "description": "short description"
+              }
+            }
+          }
+        },
+        "liquidTransportCoefficientRedistributionGenerated": {
+          "type": "boolean"
+        },
+        "thermalConductivityMoistureDependent": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "waterContent": {
+                "$ref": "number.json#/$defs/squareMeterPerSecond",
+                "description": "short description"
+              },
+              "thermalConductivity": {
+                "$ref": "number.json#/$defs/wattPerMeterKelvin",
+                "description": "short description"
+              }
+            }
+          }
+        },
+        "thermalConductivityMoistureDependentGenerated": {
+          "type": "boolean"
+        },
+        "thermalConductivityTemperatureDependent": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "temperature": {
+                "$ref": "number.json#/$defs/degreeCelcius",
```

Comment:
Would Kelvin be acceptable? We use Kelvin in the rest of the schemas and it can easily be converted.

Reply:
sure. we can use Kelvin